data/src/main/java/com/architect/coders/mu8/data/service/MarvelServiceAPI.kt

```diff
@@ -35,4 +36,11 @@ interface MarvelServiceAPI {
         @Query(OFFSET) offset: Int,
         @Query(LIMIT) limit: Int
     ): Response<BaseResponse<CharactersResponse>>
-}
\ No newline at end of file
+
+    @GET("/v1/public/events")
+    suspend fun getAllEvents(
+        @Query(TIME_STAMP) timeStamp: String,
+        @Query(API_KEY) apiKey: String,
+        @Query(HASH) hash: String
```

Comment:
Please also use the offset and limit properties to get a good number of events and not the default one.

Reply:
Done